app/models/facility_business_identifier.rb

```diff
@@ -0,0 +1,12 @@
+class FacilityBusinessIdentifier < ApplicationRecord
+  belongs_to :facility
+
+  enum identifier_type: {
+    dhis2_org_unit_id: "dhis2_org_unit_id"
+  }
+
+  validates :identifier, presence: true
+  validates :identifier_type, presence: true
+  validates :identifier_type, uniqueness: {scope: :facility_id}
+  validates :facility, presence: true
```

Comment:
I'm not sure if you truly need this, given that the relationship is not optional, but it's not hurting anything.

Reply:
This ensures you can't create a business identifier unless the facility actually exists